ospfd/ospf_zebra.c

```diff
@@ -759,30 +759,17 @@ int ospf_redistribute_unset(struct ospf *ospf, int type,
 int ospf_redistribute_default_set(struct ospf *ospf, int originate, int mtype,
 				  int mvalue)
 {
-	struct ospf_redist *red;
-
 	ospf->default_originate = originate;
 
-	red = ospf_redist_add(ospf, DEFAULT_ROUTE, 0);
```

Comment:
Scrolling through the code here I see we were calling `ospf_redist_add` twice before (???), after this PR it's called once in the VTY code but the `ospf_redist_del` is still in `ospf_zebra.c`.  Can we make this symmetric?  (Preferably with only UI stuff in `ospf_vty.c`)

Reply:
Moved to `ospf_vty.c`.